package.json

```diff
@@ -89,20 +95,19 @@
     "@types/jest-axe": "3.2.2",
     "@types/node": "10.17.55",
     "@types/prettier": "1.19.1",
+    "@types/react": "16.9.42",
+    "@types/react-dom": "16.9.10",
+    "@types/tmp": "0.2.0",
     "@types/webpack-dev-middleware": "4.1.0",
     "@types/webpack-env": "1.16.0",
-    "@types/chalk": "2.2.0",
     "@types/yargs": "13.0.11",
-    "@types/tmp": "0.2.0",
-    "@types/react": "16.9.42",
-    "@types/react-dom": "16.9.10",
     "beachball": "1.53.1",
+    "chalk": "4.1.0",
```

Comment:
bumped chalk to 4.1.0 
-> after I added additional `@nrwl/*` packages (they have chalk 4.1 as dependency), 
   build started to fail  
-> because react-conformance still depends on old chalk 2 ( which did lot of mistakes regarding exposing APIs for js and ts consumers), so `scripts/exec` was loading chalk 4.1 but `react-conformance` was supposed to consume 2.x but because you cannot consume 2 versions of the same package without hacks (aliasing), react-conformance got v4 which has different API and thus failed build